.gitignore

```diff
@@ -32,6 +32,7 @@ yarn-error.log*
 .env.development.local
 .env.test.local
 .env.production.local
+*.env
```

Comment:
This is fine. Looks like you could also just modify line 30 to be `*.env`, it'll also ignore `.env`.